public/index.html

```diff
@@ -10,6 +10,6 @@
     <div id="root"></div>
 
     <!-- Scripts -->
-    <script type="text/javascript" src="//maps.googleapis.com/maps/api/js?libraries=places"></script>
+    <script type="text/javascript" src="//maps.googleapis.com/maps/api/js?libraries=places,geometry"></script>
```

Comment:
Stupid question, but is it possible to load this library through an import? (https://github.com/google/google-api-nodejs-client)